packages/ui/src/components/common/Wrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+
+export const Row = styled.div`
+  display: flex;
+`;
+
+export const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
```

Comment:
```suggestion
export const Flex = styled.div<{
  direction?: string;
  justify?: string;
  align?: string;
  gap?: number;
  width?: number;
  height?: number;
}>`
  display: flex;
  flex-direction: ${({ direction }) => (direction ? `${direction}` : 'row')};
  justify-content: ${({ justify }) => (justify ? `${justify}` : 'center')};
  align-items: ${({ align }) => (align ? `${align}` : 'center')};
  gap: {({ gap }) => (gap ? `${gap}px` : '0px')};

  width: ${({ width }) => (width ? `${width}px` : '100%')};
  height: ${({ height }) => (height ? `${height}px` : '100%')};
`;
`;
```

약간 이런 느낌으루,, Flex로 통일시키고 필요한 속성들 prop으로 받을 수 있게 해서 사용하는 것도 조을 거 가타융...

Reply:
반영해놓겠습니다~!